packages/circuit-ui/components/Sidebar/Sidebar.js

```diff
@@ -21,14 +21,14 @@ import { TrackingElement } from '@sumup/collector';
 
 import { useClickEvent } from '../../hooks/useClickEvent';
 
-import Header from './components/Header';
-import Footer from './components/Footer';
-import NavList from './components/NavList';
-import NavItem from './components/NavItem';
-import Backdrop from './components/Backdrop';
-import CloseButton from './components/CloseButton';
-import Aggregator from './components/Aggregator';
-import Separator from './components/Separator';
+import { Aggregator } from './components/Aggregator';
```

Comment:
Do we also want to migrate this one to TS?

(on the other hand if the motivation to migrate the others was just linting issues and it works like this, let's not 🙈)

Reply:
> (on the other hand if the motivation to migrate the others was just linting issues and it works like this, let's not 🙈)

You guessed it.